### PR TITLE
Update home page for grid-website

### PR DESCRIPTION
**Pull request by chenette**
- Move "Hyperledger Grid" title to banner image

- Rewrite content: Update the high-level overview; add "open source"
  and "we welcome contributors" blurbs

- Add links to other website pages, the grid repo, and "Grid in the news"
  (the hyperledger-grid category on hyperledger.org)

Other information (from the previous home page) will move to the About page
in a separate PR.

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: generator/source/index.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+
+* [See Hyperledger Grid in the
+  news](https://www.hyperledger.org/category/hyperledger-grid)
+
+Hyperledger Grid is an open source project on GitHub. We welcome contributors,
+both organizations and individuals, to help shape project direction, contribute
+ideas, provide use cases, and work on specific tools and examples.
+Please [join the conversation](/community).
 
-This Hyperledger project is in _incubation_. It was proposed to the community
```
Comment (peterschwarz) [inline generator/source/index.md]: Should we remove this indication of its status of "in incubation"?

Comment (chenette) [reply generator/source/index.md]: My commit already removes that line.  However, I was planning to move this info to the About page, under "Project Status", with a link to the Hyperledger Grid project page.

File: generator/source/index.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
 permalink: /
 feature-img: "images/hero-bg.jpg"
 # Copyright (c) 2018 Bitwise IO, Inc.
 # Licensed under Creative Commons Attribution 4.0 International License
 # https://creativecommons.org/licenses/by/4.0/
 ---
 
-# Hyperledger Grid
+Hyperledger Grid is a platform for building supply chain solutions that include
+distributed ledger components. It provides a set of libraries, data models, and
```
Comment (davececchi) [inline generator/source/index.md]: Also business logic (via smart contracts)

File: generator/source/index.md [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
+SDKs for developing supply chain smart contracts and client interfaces.
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+This project's goal is to accelerate development of modular blockchain-based
+solutions for cross-industry supply chain problems. Hyperledger Grid provides
+domain-specific data models, such as GS1 product definitions that can be used in
+a product catalog, example smart contracts, and other modular components.
```
Comment (davececchi) [inline generator/source/index.md]: This feels redundant and I'd just combine it with the above.

File: generator/source/index.md (hunk 1 above)
Comment (chenette) [reply generator/source/index.md]: I restored this paragraph to the home page.

File: generator/source/index.md (hunk 2 above)
Comment (chenette) [reply generator/source/index.md]: Added.

File: generator/source/index.md (hunk 3 above)
Comment (chenette) [reply generator/source/index.md]: @davececchi - I reworked the first paragraph to include the non-duplicate content from the second paragraph. Is the new version any better?

**Pull request merged**